web3/providers/ipc.py

```diff
@@ -100,7 +100,7 @@ def get_default_ipc_path() -> str:  # type: ignore
             return ipc_path
 
         base_trinity_path = Path('~').expanduser() / '.local' / 'share' / 'trinity'
-        ipc_path = str(base_trinity_path / 'mainnet' / 'jsonrpc.ipc')
```

Comment:
Question: should this perhaps remain, for compatibility with very old versions of Trinity?

Reply:
No, at this point we're still in alpha and no need to preserve support for old versions since this can be done manually if people really need to connect to an old version.